aioslimproto/display.py

```diff
@@ -112,7 +112,7 @@ def __init__(
         self.width = width
         self.visualisation_type = visualisation_type
         # NOTE: height can only be 32, its not adjustable
-        self.image = Image.new("1", (width, 32))
+        self.image = Image.new("1", (self.width, 32))
```

Comment:
```suggestion
        self.image = Image.new("1", (width, 32))
```

prefer the local var (as you already have it)